src/test/java/core/basesyntax/files/FileWriterImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.files;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String fileName = "temp.csv";
+    private static final String wrongName = "src/temp/temp.csv";
+    private static final Path path = Paths.get(fileName);
```

Comment:
make it local, you can't do this right here, it's a bad practice to call such logic during initialization of field

Reply:
:)